client/dashboard/task-list/index.js

```diff
@@ -6,47 +6,41 @@ import { __ } from '@wordpress/i18n';
 import { Component, Fragment } from '@wordpress/element';
 import { filter, get } from 'lodash';
 import { compose } from '@wordpress/compose';
+import classNames from 'classnames';
+import { Snackbar, Icon, Button } from '@wordpress/components';
+import { withDispatch } from '@wordpress/data';
 
 /**
  * WooCommerce dependencies
  */
-import { Card, List } from '@woocommerce/components';
+import { Card, List, MenuItem, EllipsisMenu } from '@woocommerce/components';
 import { updateQueryString } from '@woocommerce/navigation';
-import { getSetting } from '@woocommerce/wc-admin-settings';
 
 /**
  * Internal depdencies
  */
 import './style.scss';
-import Appearance from './tasks/appearance';
-import Connect from './tasks/connect';
-import Products from './tasks/products';
-import Shipping from './tasks/shipping';
-import Tax from './tasks/tax';
-import Payments from './tasks/payments';
 import withSelect from 'wc-api/with-select';
 import { recordEvent } from 'lib/tracks';
-
-const {
-	customLogo,
-	hasHomepage,
-	hasPhysicalProducts,
-	hasProducts,
-	shippingZonesCount,
-} = getSetting( 'onboarding', {
-	customLogo: '',
-	hasHomePage: false,
-	hasPhysicalProducts: false,
-	hasProducts: false,
-	shippingZonesCount: 0,
-} );
+import { getTasks } from './tasks';
 
 class TaskDashboard extends Component {
+	constructor( props ) {
+		super( props );
+		this.state = {
+			promptShown: props.promptShown,
+		};
+	}
+
 	componentDidMount() {
 		document.body.classList.add( 'woocommerce-onboarding' );
 		document.body.classList.add( 'woocommerce-task-dashboard__body' );
 
 		this.recordEvent();
+
+		if ( ! this.state.promptShown ) {
+			this.markPromptShown();
```

Comment:
So once this is shown once, this is no longer shown again?  I thought it would appear after all tasks were completed or at least persist until an option was chosen.

But I may be misreading the designs or have missed a conversation regarding this.

Reply:
My understanding from the design was that the prompt would show the first time ("we'll show this to the user next time they log in") and the remove card button would be available for hiding the card when you return in the future.

However, it's not super clear to me if my understanding is correct or if you are correct. I can remove this functionality for now so the message persists until an option is chosen, and we can adjust in a follow-up if needed once @jameskoster is back from AFK.

Reply:
Changes this to persist until dismissed in https://github.com/woocommerce/woocommerce-admin/pull/3021/commits/50b40260cb5c93bdfcb532d6ae891f26ab663d32. Added a note on my todo list to ping Jay next week.

Reply:
Ya, I see what you mean; I can see this going either way.  Let's merge as is and we can update based on @jameskoster's response when he's back.

Reply:
Persistent is good :)